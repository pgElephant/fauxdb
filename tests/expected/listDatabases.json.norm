{
  "databases": [
    {
      "empty": false,
      "name": "test",
      "sizeOnDisk": 1024
    }
  ],
  "ok": 1,
  "totalSize": 1024
}
