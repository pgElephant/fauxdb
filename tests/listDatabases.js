db.runCommand({listDatabases: 1})
