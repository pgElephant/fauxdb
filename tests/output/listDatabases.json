{
  '0': { name: 'test', sizeOnDisk: Long('1024'), empty: false },
  ok: 1,
  databases: [],
  totalSize: Long('1024'),
  totalSizeMb: Long('1')
}
