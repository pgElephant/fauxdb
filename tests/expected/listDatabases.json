{
  "ok": 1,
  "databases": [
    {
      "name": "test",
      "sizeOnDisk": 1024,
      "empty": false
    }
  ],
  "totalSize": 1024
}
